### DISCUSSION TITLE
公式环境中相同两个字母之间空隙过大

### DISCUSSION THREAD

**Discussion by real-bird-kun** [tuna/thuthesis#749] [Q&A] [answered]
## 编译环境
编译的系统：Windows 10 
TeX 发行版：TeX Live 2021
模板版本：v7.2.4
模板类型： master 

## 描述问题
我在数学环境中有两个相同的字母f，个人觉得f与f之间的间距有点大，不是很好看。想问问有什么解决办法。

截图：

![image](https://user-images.githubusercontent.com/31504274/159948177-8b9cb500-74c6-4df7-8bc6-77940cd358f5.png)

复现上述问题的代码：
```TeX
\begin{equation}
  \label{eq:diffd}
  diff\_d = (windist\_o)_{cs} - (windist\_o)_{ls} 
\end{equation}
```

**Comment by zepinglee** [comment, marked as answer]
```tex
\begin{equation}
  \label{eq:diffd}
  \mathit{diff\_d} = (\mathit{windist\_o})_{cs} - (\mathit{windist\_o})_{ls}
\end{equation}
```
<img width="326" alt="Screen Shot 2022-03-24 at 23 42 49" src="https://user-images.githubusercontent.com/12290822/159954923-d83abc78-453f-4d63-ad78-28a8646709c2.png">

**Comment by Harry-Chen** [comment]
需要提醒这并非模板或 LaTeX 的问题，在公式环境中直接书写文字本身就是不正确的。

**Reply by real-bird-kun** [reply]
好的，谢谢您的帮助: )
我后面会多加注意的。可以考虑把这个小建议放进thuthsis-example.tex文档。
